scripts/setup-offline/setup-stt.js

```diff
@@ -10,23 +10,23 @@ import os from '@/helpers/os'
 export default () => new Promise(async (resolve, reject) => {
   log.info('Setting up offline speech-to-text...')
 
-  const destDeepSpeechFolder = 'bin/deepspeech'
+  const destCoquiFolder = 'bin/coqui'
   const tmpDir = 'scripts/tmp'
-  const deepSpeechVersion = '0.9.3'
+  const coquiVersion = '1.0.0'
```

Comment:
`s/1.0.0/1.1.0/`

Reply:
i couldn't find a `v1.1.0` here: https://github.com/coqui-ai/STT-models/tree/main/english/coqui, so i've renamed the to `coquiModelVersion` to distinguish it from the release version.